packages/cli/src/util/deploy/process-deployment.ts

```diff
@@ -97,8 +97,7 @@ export default async function processDeployment({
     vercelOutputDir,
     rootDirectory,
     skipAutoDetectionConfirmation,
-    archive:
-      process.env.VERCEL_SPLIT_ARCHIVE && archive ? 'split-tgz' : archive,
+    archive: archive,
```

Comment:
Just to clarify, does this remove the env var support completely?

Reply:
I was going to say that env var would make it easier to feature flag, but for CLI running locally (which would be the case for `--archive`) we don't really have access to feature flag env vars in the same way as for deployments, so SGTM.

Reply:
```suggestion
    archive,
```